configure/os/CONFIG_SITE.Common.RTEMS

```diff
@@ -16,8 +16,8 @@ RTEMS_VERSION = 5
 RTEMS_BASE = /home/travis/.rtems
 
 # FHI:
-#RTEMS_VERSION = 5
-#RTEMS_BASE = /home/h1/DBG/rtems
+RTEMS_VERSION = 5
+RTEMS_BASE = /opt/RTEMS/qoriq/rtems/$(RTEMS_VERSION)
```

Comment:
Does it make sense to set RTEMS_VERSION and RTEMS_BASE here?  I should comment out these lines right?

Reply:
It has been on my TODO list to see if these could simply be moved to `configure/CONFIG_SITE.local`.

Reply:
I've been setting them in CONFIG_SITE.local in my local repo without issue so far.

Reply:
CONFIG_SITE.local gets pulled in by CONFIG_SITE which is the first such file to be included so yes, these can be set there, but will be overridden by any values that later get set in this file. Historically my APS Jenkins jobs have never had to configure them because the repo has always defaulted to our settings, but I've just fixed that (I think).

Reply:
Does this mean that the "defaults" in `configure/os/CONFIG_SITE.Common.RTEMS` can be commented out now?

Reply:
Yes, I updated my Jenkins job configurations to set them itself.